### PR TITLE
mobile menu border color will follow link color from settings

### PR DESCRIPTION
**Pull request by retif**
**Related Issue/Ticket:**

https://3.basecamp.com/3608022/buckets/17045978/todos/3329831458

## Steps to test:

- [ ] change light link settings in admin panel
- [ ] check that mobile menu border color changes to the same color as link has

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

**Comment by retif** [comment]
retest

File: js/gated-content/src/scss/components/_top-menu.scss [hunk 1]
```diff
@@ -118,17 +118,16 @@
...
         a {
-          border:      none;
-          padding:     15px 10px;
-          width:       100%;
-          font-family: Verdana, sans-serif;
+          border:        1px solid $gray;
+          border-radius: 5px;
+          padding:       15px 10px;
+          width:         100%;
+          font-family:   Verdana, sans-serif;
```
Comment (anpolimus) [inline js/gated-content/src/scss/components/_top-menu.scss:130]: Do we need to use variable here?

**Review by anpolimus — CHANGES_REQUESTED** [review]

**Review by anpolimus — APPROVED** [review]

**Review by NadyaRudenko — APPROVED** [review]
tested, all good. thanks, @retif !

**Comment by retif** [comment]
retest

**Pull request merged**